src/encoded/schemas/tissue.json

```diff
@@ -93,6 +101,22 @@
             "minimum": 0,
             "maximum": 14
         },
+        "preservation_medium": {
+            "title": "Preservation Medium",
+            "description": "Medium used for tissue preservation",
+            "type": "string"
+        },
+        "preservation_type": {
+            "title": "Preservation Type",
+            "description": "Method of tissue preservation",
+            "type": "string",
+            "enum": [
+                "Fixed",
+                "Fresh",
+                "Frozen",
+                "Snap Frozen"
```

Comment:
These fields are duplicated - please ref the one from sample or add as a mixin

Reply:
TPC requested them in Tissue in addition to TissueSample. The thought was that having them in both would allow validation with GCC submissions of TissueSample.
I will ref the one from sample here.